doc/HOWTO/keys.txt

```diff
@@ -68,5 +68,41 @@ you don't want your key to be protected by a password, remove the flag
     type in the password every time the server needs to access
     the key.
 
+
+4. To generate an EC key
+
+An EC key can be used both for key agreement (ECDH) and signing (ECDSA).
+
+Generating a key for ECC is similar to generating a DSA key, which is also
+a two-stup process. First, you have to generate EC parameters from which to
+generate the key:
+
+  openssl ecparam -name secp256k1 -out secp256k1.pem
+
+The secp256k1 (stands for 'SECG curve over a 256 bit prime field') is the
```

Comment:
If intention is to steer people to most secure and high-performance option, then prime256v1 would make a better example.